### ISSUE TITLE
Tables incorrectly baseline-align to first row

### ISSUE THREAD

**Issue by digama0** [mathjax/MathJax#891]
Here is a picture of the "[TeX Samples](http://www.mathjax.org/demos/tex-samples/)" page on the MathJax homepage:
![capture3](https://cloud.githubusercontent.com/assets/868588/3948267/3152b9dc-26a2-11e4-8479-0d0f3b8bff4c.PNG)

In case it's not obvious, the issue I am referring to is the fact that the table is aligned to the first row, so there is a large white space above the table in the brackets. I can't imagine that this is expected behavior, and if it is I can't figure out what TeX to write to work around this issue. Any ideas as to what is causing this? (Apologies if this is a dupe, I didn't see anything similar.) I believe that the TeX code to generate this image is:

```
\mathbf{V}_1 \times \mathbf{V}_2 =  \begin{vmatrix}
\mathbf{i} & \mathbf{j} & \mathbf{k} \\
\frac{\partial X}{\partial u} &  \frac{\partial Y}{\partial u} & 0 \\
\frac{\partial X}{\partial v} &  \frac{\partial Y}{\partial v} & 0
\end{vmatrix}
```

**Comment by dpvc** [comment]
I can't reproduce the problem in Firefox 34.0.5 or 35, so am removing the test wanted label.
